datahub-upgrade/src/main/java/com/linkedin/datahub/upgrade/nocode/CreateAspectTableStep.java

```diff
@@ -29,18 +36,45 @@ public int retryCount() {
   @Override
   public Function<UpgradeContext, UpgradeStepResult> executable() {
     return (context) -> {
+
+      DbType targetDbType = context.parsedArgs().containsKey(DB_TYPE_ARG)
+          ? DbType.valueOf(context.parsedArgs().get(DB_TYPE_ARG).get())
+          : DbType.MYSQL;
+
+      String sqlUpdateStr;
+
+      switch (targetDbType) {
+        case POSTGRES:
+          sqlUpdateStr = "CREATE TABLE IF NOT EXISTS metadata_aspect_v2 (\n"
+              + "  urn                           varchar(500) not null,\n"
+              + "  aspect                        varchar(200) not null,\n"
+              + "  version                       bigint(20) not null,\n"
```

Comment:
After some testing, it seems that `bigint(20)`  is not supported in Postgres. The equivalent is simply `bigint`, it seems.

Reply:
Yes, you're right. Just also verified. Thanks!